React-Template-master/src/components/MessageModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Modal, Box, Typography, Button, useTheme } from '@mui/material';
+import { NEUTRAL } from '../theme/palette';
+import { CorrectLogo, WarningLogo } from './Logo';
+import { useTranslation } from 'react-i18next';
+
+interface ConfirmationModalProps {
+    open: boolean;
+    setOpen: Function;
+    modalContent: {
+        title: string;
+        subtitle: string;
+        type?: string;
```

Comment:
Create an enum or type for the type of the model content, string is way too generalized

Reply:
okay